lib/v2/patagonia/new-arrivals.js

```diff
@@ -0,0 +1,53 @@
+const got = require('@/utils/got');
+const { art } = require('@/utils/render');
+const path = require('path');
+const cheerio = require('cheerio');
+const host = 'https://www.patagonia.com';
+const categoryMap = {
+    mens: ['mens-new', 'mens-new-arrivals'],
+    womens: ['womens-new', 'womens-new-arrivals'],
+    kids: ['kids-new-arrivals', 'kids-baby-new-arrivals'],
+    luggage: ['luggage-new-arrivals', 'luggage-new-arrivals'],
+};
+module.exports = async (ctx) => {
+    const { category } = ctx.params;
+    const url = `${host}/on/demandware.store/Sites-patagonia-us-Site/en_US/Search-LazyGrid`;
+    const response = await got({
+        method: 'get',
+        url,
+        searchParams: {
+            cgid: categoryMap[category][0],
+            isLazyGrid: true,
+        },
+    });
+    const data = response.data;
+
+    const $ = cheerio.load(data);
+    const list = $('.product')
+        .map(function () {
+            const data = {};
+            data.title = JSON.parse($(this).find('.product-tile').attr('data-tealium')).product_name[0];
```

Comment:
```suggestion
            data.title = $(this).find('.product-tile').data('tealium').product_name[0];
```
https://github.com/DIYgod/RSSHub/pull/12045#discussion_r1126306223